Source/ACE/Entity/Landblock.cs

```diff
@@ -276,7 +276,7 @@ public List<WorldObject> GetWorldObjectsInRange(WorldObject wo, float maxrange,
             return allworldobj;
         }
 
-        public List<WorldObject> GetWorldObjectsByGuid(ObjectGuid objectguid, bool neighbors)
+        private List<WorldObject> GetWorldObjectsByGuid(ObjectGuid objectguid, bool neighbors)
         {
```

Comment:
1) Guid is unique.  Or it is designed to be so, anyway.  Duplicate guids = big bug.  Returning a List here is a waste.
2) Locking just to copy the entire list is a waste.  Check for the item in the lock.
3) If you find it here, there's no reason to hit the adjacencies.
4) If you find it in an adjacency, either return or continue out of the loop to stop looking.